### ISSUE TITLE
Better highlighting

### ISSUE THREAD

**Issue by hrach** [juzna/intellij-neon#36]
Would be great if better highlighter was implemented. Possible features:

- highlight values
- highlight `%parameters%`
- highlight class References

**Comment by matej21** [comment]
Thanks for the suggestion. The issue is that neither references nor parameters are a part of neon language (not sure what do you mean by "value"). 

I'm considering implementing another language for a language injection. It will handle all the nette-di specific features, like `@`, '%parameter%` etc.

**Comment by hrach** [comment]
>  The issue is that neither references nor parameters are a part of neon language

Of course, but neon sadly force class completion everywhere, also in parameters section, and this is also part of the Nette Framework support.

The value is everyrhing after `:`. and not other type of token. Only possibility is change the general / text / default text color.
